src/components/EventCardsGridContainer/index.js

```diff
@@ -39,7 +39,7 @@ const messages = defineMessages({
 });
 
 const { Pending } = TransactionStatus;
-const LIMIT = 8;
+const LIMIT = 16;
```

Comment:
lets change this to a higher number so it covers all screen resolutions. lets do 50